src/sparseml/pytorch/optim/modifier.py

```diff
@@ -115,18 +127,80 @@ def loggers(self):
         """
         return self._loggers if self._loggers is not None else []
 
-    def initialize(self, module: Module, optimizer: Optimizer):
+    def state_dict(self) -> Dict[str, Dict]:
+        """
+        :return: PyTorch state dictionary to store any variables from this modifier
+        """
+        return {}
+
+    def load_state_dict(self, state_dict: Dict[str, Dict], strict: bool = True):
         """
-        Handles initializing and setting up the modifier.
-        Called once on construction of the scheduled optimizer.
+        Loads the given state dict into this modifier
 
-        :param module: module to modify
-        :param optimizer: optimizer to modify
+        :param state_dict: dictionary object as generated by this object's state_dict
+            function
+        :param strict: True to raise a KeyError for any missing or extra information in
+            the state dict, False to ignore
+        :raises IndexError: If any keys in the state dict do not correspond to a valid
+            index for this manager and strict=True
+        """
+        pass
```

Comment:
instead of `pass` I think the minimal functionality here should be checking for `strict` and raising an index error if anything is present.

Reply:
good point, fixing